assets/package.json

```diff
@@ -7,22 +7,26 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "shresthagrawal.31@gmail.com",
+  "contributors": [
+    "prateek@marlin.pro"
+  ],
   "license": "ISC",
   "dependencies": {
-    "libp2p": "^0.25.4",
-    "libp2p-bootstrap": "^0.9.7",
-    "libp2p-kad-dht": "^0.15.0",
-    "libp2p-mdns": "^0.12.3",
-    "libp2p-secio": "^0.11.1",
-    "libp2p-tcp": "^0.13.0",
-    "libp2p-webrtc-star": "^0.16.1",
-    "libp2p-websockets": "^0.12.2",
-    "multiaddr": "^6.1.0",
-    "peer-info": "^0.15.1",
+    "it-pipe": "^1.1.0",
+    "libp2p": "^0.27.6",
+    "libp2p-bootstrap": "^0.10.4",
+    "libp2p-gossipsub": "^0.3.0",
+    "libp2p-kad-dht": "^0.18.6",
+    "libp2p-mdns": "^0.13.3",
+    "libp2p-mplex": "^0.9.5",
+    "libp2p-secio": "^0.12.4",
+    "libp2p-tcp": "^0.14.4",
+    "libp2p-webrtc-star": "^0.17.9",
+    "libp2p-websockets": "^0.13.6",
+    "multiaddr": "^7.4.3",
+    "peer-info": "^0.17.5",
     "promisify-es6": "^1.0.3",
     "protons": "^1.0.1",
-    "pull-mplex": "^0.1.2",
-    "pull-stream": "^3.6.11",
-    "wrtc": "^0.4.1"
+    "wrtc": "^0.4.4"
```

Comment:
Can we bump this to 0.4.6?  I'm getting npm install errors for this package with 0.4.4